contracts/MultiPartyEscrow.sol

```diff
@@ -100,10 +101,11 @@ contract MultiPartyEscrow {
     private
     {
         PaymentChannel storage channel = channels[channelId];
-        balances[channel.sender]      += channel.value; 
-        channel.value                  = 0;
-        channel.nonce                 += 1;
-        channel.expiration             = 0;
+
+        balances[channel.sender] = balances[channel.sender].add(channel.value); 
+        channel.value            = 0;
+        channel.nonce           += 1;
```

Comment:
No, It doesn't make sense here. 
1. "1" doesn't come from outside word, we shoudn't check it.
2.  overflow wouldn't be an issue here (even if we could overflow with +1, uint256), because logic of the channel will work...

Reply:
I want to say it again, but I'm feeling rather strange arguing that we shouldn't make extra checks, because I usually teach people to always make extra checks. But let's not forget that here we pay real money for each operation. And adding extra cost only to be "consistent" is really really strange.
I want to underline again, MPE is very small contract, which will be used the most in SingularityNet, so our consumers will pay most of their gas here.

We need to be consistent only in the situation where we might forget to make checks in important places. But here it is not the case.
We already using SafeMath in places which is direct double check, I really don't see the reason why you want to push it to an extreme.